tests/test_dbclient.py

```diff
@@ -59,30 +59,30 @@ def reset(db_client, data_provider_list):
     )
 
 # Commenting out tests now due to nondeterministic bugs.
```

Comment:
Ok so -are these tests now going to always pass? Because the TODOs about nondeterministic bugs are still there :o

Reply:
Forgot to delete comment, deleted.

I ran the tests like 10 times and they passed each time so I'm just hoping I didn't get baited and that the bug is gone for good.